clients/cli/src/main/java/org/apache/gravitino/cli/commands/MetalakeEnable.java

```diff
@@ -69,4 +69,10 @@ public void handle() {
 
     System.out.println(msgBuilder);
   }
+
+  @Override
+  public Command verify() {
+    if (metalake == null) exitWithError(ErrorMessages.MISSING_METALAKE);
+    return super.verify();
+  }
```

Comment:
You don't need to duplicate this method several times. Also check if it is actually needed, as we don't want to duplicate work the CLI or other code does.